app/models/notification/SDESNotificationFile.scala

```diff
@@ -24,7 +24,8 @@ case class SDESNotificationFile(
                                  location: String,
                                  checksum: SDESChecksum,
                                  size: Int,
-                                 properties: Seq[SDESProperties]
+                                 caseId: SDESProperties,
+                                 sourceFileUploadDate: SDESProperties
```

Comment:
This would probably fail when being sent to SDES as it does not conform to spec, we should have a `writes` that is specific for auditing only, rather than changing the model 😃